snap/README.md

```diff
@@ -64,6 +65,14 @@ The path to the runtime provided by the Kata Containers snap image is
 `/snap/kata-containers/current/usr/bin/kata-runtime`. You should use it to
 run Kata Containers with [docker][9] and [kubernetes][10].
 
+## Remove snap
+
+You can remove Kata Containers by running the following command:
```

Comment:
s/Kata Containers/the Kata Containers snap/

Reply:
done.